vendor/bat-native-ads/src/bat/ads/internal/segments/segments_util.cc

```diff
@@ -122,6 +131,11 @@ bool ShouldFilterSegment(const std::string& segment) {
   const auto iter = std::find_if(
       filtered_segments.begin(), filtered_segments.end(),
       [&segment](const FilteredCategoryInfo& filtered_segment) {
+        if (filtered_segment.name.empty()) {
+          BLOG(0, "Ads filtered category shouldn't be empty.");
+          return false;
+        }
+
```

Comment:
`filtered_segment.name` should never be empty. Can we please instead add DCHECKs and trace the callers of this method to understand why `filtered_segment.name` could be empty